Makefile

```diff
@@ -48,14 +47,17 @@ ifndef SKIP_SERVER_RUST  # Skip building aw-server-rust if SKIP_SERVER_RUST is d
 	else \
 		echo 'Rust not found, skipping aw-server-rust!'; \
 	fi
+else
+	make --directory=aw-server build SKIP_WEBUI=$(SKIP_WEBUI)
```

Comment:
This will break CI, which should build both (and building both should be the default).